addons/web/__manifest__.py

```diff
@@ -211,8 +211,9 @@
             ('include', 'web._assets_bootstrap'),
 
             'web/static/src/env.js',
-            'web/static/src/core/utils/transitions.scss', # included early because used by other files
+            'web/static/src/core/utils/transitions.scss',  # included early because used by other files
```

Comment:
Probably don't want this whitespace-only change

Reply:
Probably need to install a linter by default for each developer.
Since we touch 2 lines after, not shocking imo.

Reply:
Indeed, fixing linter error/typo when 1-2 lines above/below if fine IMHO